client/src/components/forms/LoginForm.tsx

```diff
@@ -0,0 +1,89 @@
+import * as React from 'react';
+import Button, { FilledInput, IconButton, InputAdornment, InputLabel } from '@mui/material';
+import { Label, Visibility, VisibilityOff } from '@mui/icons-material';
+import TextField from '@mui/material/TextField';
+import {post} from '../../axios.service';
+import { AxiosError, AxiosResponse } from 'axios';
+
+interface LoginInfo {
+    username: string;
+    password: string;
+    showPassword: boolean;
+}
+
+const LoginForm = () => {
+
+    const [values, setValues] = React.useState<LoginInfo>({
+        username: '',
+        password: '',
+        showPassword: false,
+    });
+
+    const [displayError, setDisplayError] = React.useState<boolean>(false);
+    
+    const handleChange =
+        (prop: keyof LoginInfo) => (event: React.ChangeEvent<HTMLInputElement>) => {
+        setValues({ ...values, [prop]: event.target.value });
+        };
+    
+    const handleClickShowPassword = () => {
+        setValues({
+        ...values,
+        showPassword: !values.showPassword,
+        });
+    };
+    
+    const handleMouseDownPassword = (event: React.MouseEvent<HTMLButtonElement>) => {
+        event.preventDefault();
+    };
+
+    const handleSubmit = () => {
+        const response = post('/auth/login', 
+            {
+			    username : values.username,
+                password : values.password
+		    }, {}, (response: AxiosResponse) => {
```

Comment:
You have lambda functions that do nothing. Weren't they supposed to do stuff like close the page and save the token?